etna/analysis/outliers/prediction_interval_outliers.py

```diff
@@ -37,8 +37,34 @@ def create_ts_by_column(ts: "TSDataset", column: str) -> "TSDataset":
     return TSDataset(new_df, freq=ts.freq)
 
 
+def select_segments_subset(ts: TSDataset, segments: List[str]) -> TSDataset:
```

Comment:
May be we should emphasize that this function is private by giving it `_` in the beginning.